src/TextInput/FormLabel.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import * as React from 'react';
+import { View, Text } from 'react-native';
+
+import { StyleSheet } from '..';
+import defaultTokens from '../defaultTokens';
+
+type Props = {|
+  +children: React$Node,
+  +filled?: boolean,
+  +disabled?: boolean,
+  +required?: boolean,
+  +inlineLabel?: boolean,
+|};
+
+const getAsteriksColor = filled =>
+  !filled
+    ? defaultTokens.orbit.colorTextError
+    : defaultTokens.orbit.colorFormLabelFilled;
+
+const getColor = (filled, disabled) =>
+  !filled || disabled
+    ? defaultTokens.orbit.colorFormLabel
+    : defaultTokens.orbit.colorFormLabelFilled;
+
+const Asteriks = ({ filled, children }) => (
+  <Text style={[styles.asteriks, { color: getAsteriksColor(filled) }]}>
```

Comment:
I believe that means that this style will not take advantage of `StyleSheet.create`. What do you think about my approach there: https://github.com/kiwicom/universal-orbit-components/blob/7dd2afa002f089c4a898ce02f00be0147bf2687c/src/Button/ButtonTitle.js#L32